how-it-works.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
This is how it shows on iPad
![image](https://user-images.githubusercontent.com/60205816/182843071-f2154ddc-91d0-4651-826c-5fbd5de83825.png)
I think this is a result of the width of the screen.
Do well to fix it, thanks